census/src/test/scala/model/RssdToMsaSpec.scala

```diff
@@ -0,0 +1,13 @@
+package model
+
+import hmda.census.model.RssdToMsa
+import org.scalatest.{ MustMatchers, WordSpec }
+
+class RssdToMsaSpec extends WordSpec with MustMatchers {
+  "RssdToMsa" must {
+    "map values correctly" in {
```

Comment:
You could add the tests after you import the updated version of the file.
You can look at `CbsaLookupSpec` for examples